src/textual/strip.py

```diff
@@ -199,7 +199,7 @@ def join(cls, strips: Iterable[Strip | None]) -> Strip:
         return strip
 
     def __bool__(self) -> bool:
-        return bool(self._segments)
```

Comment:
This is faster!?
If so, _maybe_ it's worth adding a comment in front saying this is a micro optimisation.

What if you call `self._segments.__bool__()` yourself?

Reply:
It's faster because `bool()` requires a global lookup and a function call. `not not` is two op-codes, no explicit function call.

It's around 75% faster. But still unlikely to make a tangible difference.